notifications_utils/template.py

```diff
@@ -158,6 +178,10 @@ def __str__(self):
             str.strip
         )
 
+    @property
+    def content_replaced(self):
+        return str(self)
```

Comment:
I think a comment above this to explain why we are overriding the base method would be good here please

Reply:
Comment updated in 4d3021f